pkg/controller/embercsi/embercsi_controller.go

```diff
@@ -586,18 +585,12 @@ func getVolumes (ecsi *embercsiv1alpha1.EmberCSI, csiDriverMode string) []corev1
 				},
 			},
 		},{
-			Name: "lvm-dir",
+			Name: "var-lib-iscsi",
 			VolumeSource: corev1.VolumeSource{
 				HostPath: &corev1.HostPathVolumeSource{
-					Path: "/etc/lvm",
+					Path: "/var/lib/iscsi",
 				},
 			},
-		},{
```

Comment:
-1: This will break the LVM backend, we should add it conditionally depending on the backend.